runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -209,6 +209,7 @@ private void sendLinuxNotification(
 		executorService.submit(() ->
 		{
 			final boolean success = sendCommand(commands)
+					.filter(process -> !process.isAlive())
```

Comment:
Wouldn't this case success to be always false? As this will never wait for process to be terminated (e.g .waitFor() needs to be used before exitValue check)

Reply:
Ok I've changed the code according to your instructions.
I've tested my first solution as well and it worked. But you are probably right and calling .waitFor() is more safe.